docs/contribute/locally.md

```diff
@@ -35,7 +35,7 @@ This guide uses the first option. If you'd like to clone the repository and buil
    Run this command to download and install the latest version of `docs-builder`:
 
    ```sh
-   sudo curl -L https://raw.githubusercontent.com/elastic/docs-builder/refs/heads/main/install.sh | sh
+   sudo curl -sL https://ela.st/docs-builder-install | sh
```

Comment:
:stop_sign: If I'm honest, I'm rather concerned about this being a recommendation:

- generally as a `curl | sh`
- then also throwing in `sudo` here
- the fact that this is a short URL that could be modified to point to any random location
- not teaching Elasticians to audit the scripts they're executing with superuser privileges

Looking up this pattern generally results in many articles such as https://lukespademan.com/blog/the-dangers-of-curlbash/ and https://security.stackexchange.com/questions/213401/is-curl-something-sudo-bash-a-reasonably-safe-installation-method and https://macarthur.me/posts/curl-to-bash/

Reply:
I understand the concern (and thanks for the suggestion, I've already applied it). It's a popular critique. However:

- We need a convenient, fast way of installing and updating docs-builder until we've packages set up.
- We control the [ela.st](http://ela.st/) links. Should Elasticians not trust Elastic links? Will the same happens with packages?
- We're not asking folks to run these instructions for CI/CD.

If the concern is sudo, I think we could work around that by adding the file to a PATH that is not part of the system (/.local/bin?).

Reply:
Another option would be to distribute the install scripts with the release ZIPs and let users use them if they want.

Reply:
I've removed `sudo` from the command, as the user will be prompted interactively anyway.